android/android-espresso-demo-project/codemagic.yaml

```diff
@@ -5,27 +5,28 @@ workflows:
         name: Native Android
         instance_type: mac_pro
         environment:
+            groups:
+                - aws_credentials # <-- (Includes: AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_PROJECT_ARN, AWS_DEVICE_POOL_ARN, AWS_DEFAULT_REGION)
+                - keystore_credentials # <-- (Includes: CM_KEYSTORE, CM_KEYSTORE_PASSWORD, CM_KEY_ALIAS_PASSWORD, CM_KEY_ALIAS_USERNAME)
+                - google_play # <-- (Includes: GCLOUD_SERVICE_ACCOUNT_CREDENTIALS)
+                - other
+            # Add the group environment variables in Codemagic UI (either in Application/Team variables) - https://docs.codemagic.io/variables/environment-variable-groups/
             vars:
                 # AWS Device Farm Environment Variables
                 # See more information from official AWS Device Farm docs:
                 # https://docs.aws.amazon.com/devicefarm/latest/developerguide/how-to-create-test-run.html#how-to-create-test-run-cli
-                AWS_ACCESS_KEY_ID: Encrypted(...) # <-- Put your encrypted access key id here. See more info at https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys
-                AWS_SECRET_ACCESS_KEY: Encrypted(...) # <-- Put your encrypted secret access key here. See more info at https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys
-                AWS_PROJECT_ARN: Encrypted(...) # <-- Put your encrypted Project ARN here. Get Project ARN with `aws devicefarm list-projects`.
-                AWS_DEVICE_POOL_ARN: Encrypted(...) # <-- Put your encrypted Device Pool ARN here. Choose a Device Pool ARN using `aws devicefarm list-device-pools --arn "${AWS_PROJECT_ARN}"`.
-                AWS_DEFAULT_REGION: "us-west-2" # See available options at https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/
+                # AWS_ACCESS_KEY_ID:  # <-- Put your access key id here. See more info at https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys
+                # AWS_SECRET_ACCESS_KEY: # <-- Put your secret access key here. See more info at https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys
+                # AWS_PROJECT_ARN: # <-- Put your Project ARN here. Get Project ARN with `aws devicefarm list-projects`.
+                # AWS_DEVICE_POOL_ARN: # <-- Put your Device Pool ARN here. Choose a Device Pool ARN using `aws devicefarm list-device-pools --arn "${AWS_PROJECT_ARN}"`.
+                # AWS_DEFAULT_REGION: "us-west-2" # See available options at https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/
```

Comment:
I would move these comments to the groups section because people might think they need to put the values in the `vars` section. 

I would also remove the colon after each variable so instead of `AWS_ACCESS_KEY_ID:` it shows as `AWS_ACCESS_KEY_ID` etc.

Reply:
Done, thank you!!